eq-author-api/migrations/dropDatatypeFieldFromPipedValues.js

```diff
@@ -0,0 +1,38 @@
+//This is an auto-generated file.  Do NOT modify the method signature.
+const cheerio = require("cheerio");
+const { flatMap } = require("lodash");
+
+module.exports = function dropDatatypeFieldFromPipedValues(questionnaire) {
+  const pages = flatMap(questionnaire.sections, section =>
+    flatMap(section.pages, page => page)
+  );
+
+  pages.map(page => {
```

Comment:
It is also possible to pipe values into section introduction pages. We would need to also deal with updating these values.

Reply:
Sorry, I was not aware of that. It should be fixed now.